chia/full_node/mempool.py

```diff
@@ -368,7 +368,7 @@ def remove_from_pool(self, items: List[bytes32], reason: MempoolRemoveReason) ->
 
         return MempoolRemoveInfo(removed_internal_items, reason)
 
-    def add_to_pool(self, item: MempoolItem) -> MempoolAddInfo:
+    def add_to_pool(self, item: MempoolItem, conflicting_items: List[bytes32] = []) -> MempoolAddInfo:
```

Comment:
I think you need to document what this new parameter means or what its behavior is. Also, the resulting `MempoolInfo` is slightly different now. I suppose this isn't documented as is, but now seems to be a good opportunity to do so.